keyboards/nafuda/info.json

```diff
@@ -0,0 +1,20 @@
+{
+    "keyboard_name": "nafuda", 
+    "url": "https://salicylic-acid3.hatenablog.com/", 
+    "maintainer": "Salicylic_acid3", 
+    "width": 18.25, 
+    "height": 3, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"mouse", "x":1, "y":0},
+                {"label":"BS", "x":0, "y":0.5},
+                {"label":"browser", "x":2, "y":0.5},
+                {"label":"\u2191", "x":1, "y":1},
+                {"label":"\u2190", "x":0, "y":1.5},
+                {"label":"\u2192", "x":2, "y":1.5},
+                {"label":"\u2193", "x":1, "y":2}
```

Comment:
Order here should be:

Mouse, Backspace, `\u2191`, Browser, `\u2190`, `\u2193`, `\u2192`.

None of the values need to change; just the ordering.

Reply:
I fixed it.
Thank you.